vllm/v1/worker/gpu_input_batch.py

```diff
@@ -197,6 +198,9 @@ def __init__(
         # that are currently in the prefill phase.
         self.num_prompt_logprobs: dict[str, int] = {}
 
+        # To accumulate prompt logprobs tensor chunks across prefill steps.
+        self.in_progress_prompt_logprobs_cpu: dict[str, LogprobsTensors] = {}
```

Comment:
This need to be added to `remove_request`

Right now we have a memory leak if a request is aborted during a parital prefill since the `del self.in_progress_prompt_logprobs_cpu` is only called once we generate the first token for a partial prefill

Reply:
good catch, thanks